src/server/scripts/Northrend/VioletHold/boss_moragg.cpp

```diff
@@ -24,7 +24,18 @@ enum Spells
     SPELL_CORROSIVE_SALIVA                     = 54527,
     SPELL_OPTIC_LINK                           = 54396,
     SPELL_RAY_OF_PAIN                          = 54438, // NYI missing spelldifficulty
-    SPELL_RAY_OF_SUFFERING                     = 54442  // NYI missing spelldifficulty
+    SPELL_RAY_OF_SUFFERING                     = 54442,  // NYI missing spelldifficulty
```

Comment:
Compare with my comment in boss_lavanthor.cpp and notice the extra space added here (alignment with line above).